rest-api/tools/deploy_app.sh

```diff
@@ -182,7 +182,9 @@ if [[ -z "$test_request" ]];
 then
   echo "${BOLD}Test request failed to return the expected response, something may be wrong with
        the deployment${NONE}"
+  $UPDATE_TRACKER --version $VERSION --comment "Test request failed to return a valid response."
```

Comment:
Can we add something like "!!!!!!!!!!!! REQUEST FAILURE !!!!!!!!!" or whatever to make this stick out?